com.unity.netcode.gameobjects/Prototyping/Interpolator/SimpleInterpolator.cs

```diff
@@ -0,0 +1,86 @@
+using UnityEngine;
+
+namespace Unity.Netcode
+{
+    public abstract class SimpleInterpolator<T> : IInterpolator<T>
```

Comment:
What's the use case of this?

This is does not do `interpolation` (as in the mathematical term) because it's stateful. It is not `simple` at least not for me. And it's only used in tests from what I can tell.

Reply:
yeah, I had the same question in my mind. we can offer `NoInterpolator` & `BufferedInterpolator` but I don't get why we also need this `SimpleInterpolator` at all.

Reply:
deleted